### PR TITLE
we have to list up all the sub-packages

### PR DESCRIPTION
**Pull request by hiroshi-matsuda-rit**
@izziiyt @kazuma-t 
After some work arounds, I found that package list in setup.py should contain all the sub-packages.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,15 @@ def readme():
       long_description=readme(),
       url="https://github.com/WorksApplications/Sudachi",
       license="Apache-2.0",
-      packages=["sudachipy"],
+      packages=[
```
Comment (izziiyt) [inline setup.py]: @hiroshi-matsuda-rit Thanks for your adjustments.   
How about using `setuptools.find_namespace_packages` ? In this style, we need to add module name when we add new modules.

Comment (hiroshi-matsuda) [reply setup.py]: Sure, it's better. I'll replace it with that soon.

**Comment by hiroshi-matsuda-rit** [comment]
@izziiyt I applied some fixes to make 'pip install' available.

It looks earlier to use find_namespace_packages because it requires pip>=19.
I replaced it for find_packages.

I removed dynamic file loading functions because it ran while pip installing too and caused 'File Not Found' error.

Please consider to fill the long_description filed with appropriate text.
And we hope your publishing the pip install archives created by the latest setup.py. > @kazuma-t 

Actually, I just published a SudachiPy+SudachiDict package for spaCy Japanese model release from our fork repository for convenience, but I suppose that these archives should be published from owners repository.
https://github.com/megagonlabs/SudachiPy/releases/tag/v0.1.3-dict

**Comment by kazuma-t** [comment]
@izziiyt After you merge this PR, I will make `.travis.yml` to upload a package to GitHub.

**Pull request merged**